### PR TITLE
Using jenkins sonatype username and password instead of my own

### PR DESCRIPTION
**Pull request by sfc-gh-dshah**

### CODE REVIEW

File: snapshot_deploy.sh [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 export GPG_KEY_ID="Snowflake Computing"
-export LDAP_USER="$sonatype_user"
-export LDAP_PWD="$sonatype_password"
+export LDAP_USER="$INTERNAL_NEXUS_USERNAME"
```
Comment (sfc-gh-tzhang) [inline snapshot_deploy.sh:5]: could you update unshaded_snapshot_deploy.sh as well?

File: snapshot_deploy.sh [hunk 2]
```diff
@@ -47,6 +47,6 @@ echo "[Info] Sign package and deploy to staging area"
 project_version=$($THIS_DIR/scripts/get_project_info_from_pom.py $THIS_DIR/pom.xml version)
 $THIS_DIR/scripts/update_project_version.py public_pom.xml $project_version > generated_public_pom.xml
 
-mvn clean deploy ${MVN_OPTIONS[@]} -Dsnapshot-deploy
+mvn clean deploy ${MVN_OPTIONS[@]} -Dsnapshot-deploy -DskipTests=true
```
Comment (sfc-gh-tzhang) [inline snapshot_deploy.sh:50]: I think we should run the tests to make sure everything works?

**Review by sfc-gh-tzhang — COMMENTED** [review]
Thanks, I think we could remove the sonatype_user and sonatype_password inputs after this change?

File: snapshot_deploy.sh (hunk 1 above)
Comment (sfc-gh-dshah) [reply snapshot_deploy.sh:5]: Sure will update it.

File: snapshot_deploy.sh (hunk 2 above)
Comment (sfc-gh-dshah) [reply snapshot_deploy.sh:50]: Since we are validating all the tests with RTs, I thought we can skip testing with Jenkins. Wdyt ?

**Comment by sfc-gh-dshah** [comment]
I have removed user input from jenkins pipeline for mavenPushSNAPSHOT. We can do same for unshaded_snapshot pipeline.

**Comment by sfc-gh-dshah** [comment]
@sfc-gh-tzhang addressed PR comments.

File: unshaded_deploy.sh [hunk 3]
```diff
@@ -3,8 +3,8 @@
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 export GPG_KEY_ID="Snowflake Computing"
-export SONATYPE_USER="$sonatype_user"
-export SONATYPE_PWD="$sonatype_password"
+export LDAP_USER="$INTERNAL_NEXUS_USERNAME"
```
Comment (sfc-gh-tzhang) [inline unshaded_deploy.sh:6]: Not this file, we should be updating unshaded_snapshot_deploy.sh?

Comment (sfc-gh-dshah) [reply unshaded_deploy.sh]: Sorry my bad. Updated the wrong file.Fixed the issue.

File: unshaded_deploy.sh [hunk 4]
```diff
@@ -3,8 +3,8 @@
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 export GPG_KEY_ID="Snowflake Computing"
-export SONATYPE_USER="$sonatype_user"
-export SONATYPE_PWD="$sonatype_password"
+export LDAP_USER="$sonatype_user"
+export LDAP_PWD="$sonatype_password"
```
Comment (sfc-gh-tzhang) [inline unshaded_deploy.sh:7]: This needs to be reverted?

**Review by sfc-gh-tzhang — APPROVED** [review]
LGTM!

**Pull request merged**